content/ko/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md

```diff
@@ -9,16 +9,16 @@ weight: 30
 
 {{< feature-state for_k8s_version="v1.18" state="beta" >}}
 
-쿠버네티스를 사용하여 리눅스와 Windows 노드를 혼합하여 실행할 수 있으므로, 리눅스에서 실행되는 파드와 Windows에서 실행되는 파드를 혼합할 수 있다. 이 페이지는 Windows 노드를 클러스터에 등록하는 방법을 보여준다.
+쿠버네티스를 사용하여 리눅스와 윈도우 노드를 혼합하여 실행할 수 있으므로, 리눅스에서 실행되는 파드와 윈도우에서 실행되는 파드를 혼합할 수 있다. 이 페이지는 윈도우 노드를 클러스터에 등록하는 방법을 보여준다.
 
 
 
 
 ## {{% heading "prerequisites" %}}
  {{< version-check >}}
 
-* Windows 컨테이너를 호스팅하는 Windows 노드를 구성하려면
-[Windows Server 2019 라이선스](https://www.microsoft.com/en-us/cloud-platform/windows-server-pricing) 이상이 필요하다.
+* 윈도우 컨테이너를 호스팅하는 윈도우 노드를 구성하려면
+[윈도우 서버 2019 라이선스](https://www.microsoft.com/en-us/cloud-platform/windows-server-pricing) 이상이 필요하다.
```

Comment:
nit: The product name is Windows Server 2019, according to the page linked to.

Reply:
@sftim I agree. And I think it should be link to Korean page. I'll update on the next branch.